SPECS/ocaml-parsexp/ocaml-parsexp.spec

```diff
@@ -0,0 +1,130 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
```

Comment:
Could you clean-up the placement of these tags?

Reply:
Cleaned up